sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeGenerator.scala

```diff
@@ -789,6 +789,36 @@ class CodegenContext {
   }
 
   /**
+   * Wrap the generated code of expression by a function. ev,isNull and ev.value are passed
+   * by global variables
```

Comment:
Not arbitrary codes all can be wrapped by this. The codes must be only created from a row object. We should note this in the comment.